### DISCUSSION TITLE
resloved-Cradle testing Chrome automation operation error

### DISCUSSION THREAD

**Discussion by it666-png** [BAAI-Agents/Cradle#77] [General]
Hello, I discovered this project today and I am very interested. I immediately deployed and tested it, but found that there was an error after running it. The error is shown in the following figure. Please provide some guidance.

![cradle-chrome-error](https://github.com/user-attachments/assets/4720ece1-fdd5-4e37-bbd5-f52a578b858e)

**Comment by it666-png** [comment]
During the execution process, a Chrome window could pop up, but it subsequently failed

**Comment by WeihaoTan** [comment]
As shown in the logger report, it seems that you did not install the SAM model. Please follow our readme to install it.
